coderd/database/dbauthz/dbauthz.go

```diff
@@ -594,6 +597,7 @@ func (q *querier) canAssignRoles(ctx context.Context, orgID *uuid.UUID, added, r
 	}
 
 	grantedRoles := append(added, removed...)
+	notBuiltInRoles := make([]string, 0)
```

Comment:
nit: this reads kinda clunky, if the intent here is to count the custom roles then just name it `customRoles` / `customRolesMap`? Or am I missing something?